build.sbt

```diff
@@ -116,21 +116,36 @@ val V = new {
   val munit = "0.7.29"
   val coursier = "1.0.4"
   val scalacheck = "1.15.4"
-}
+  val kindProjector = "0.13.2"
+  val evilplot = "0.8.1"
 
-lazy val pprintVersion = Def.setting {
-  if (scalaVersion.value.startsWith("2.11")) "0.5.4"
-  else "0.6.0"
-}
+  val flexmark = "0.62.2"
+  val diffutils = "1.3.0"
+  val directoryWatcher = "0.15.0"
+  val undertow = "2.2.10.Final"
+  val xnio = "3.8.4.Final"
+  val slf4j = "1.7.32"
 
-lazy val fansiVersion = Def.setting {
-  if (scalaVersion.value.startsWith("2.11")) "0.2.6"
-  else "0.2.9"
-}
+  object npm {
+    val ms = "2.1.1"
+  }
+
+  def pprint(scalaV: String) =
```

Comment:
FYI these can also be defined as `val pprint = Def.setting { if (scalaVersion.value) ... }` and then used as `pprint.value`